cmd/oras/attach.go

```diff
@@ -152,11 +152,24 @@ func runAttach(opts attachOptions) error {
 				}
 				return content.Successors(ctx, fetcher, node)
 			}
+		} else if root.MediaType == ocispec.MediaTypeImageManifest {
+			graphCopyOptions.FindSuccessors = func(ctx context.Context, fetcher content.Fetcher, node ocispec.Descriptor) ([]ocispec.Descriptor, error) {
+				if content.Equal(node, root) {
+					// skip subject
+					return append(descs, ocispec.Descriptor{
```

Comment:
nit: Should we put the scratch config descriptor in a variable?

Reply:
currently it's not used anywhere else, we might move it to a variable when such need comes.

Reply:
also we can directly reference OCI scratch config once https://github.com/opencontainers/image-spec/pull/1023 once is merged